packages/console/src/App.tsx

```diff
@@ -103,6 +103,7 @@ const WrappedStoreRoutes = observer(() => {
     }
   }, [plugins]);
 
+  // biome-ignore lint/correctness/useExhaustiveDependencies: useEffect state machine
```

Comment:
Only disabling the rule manually, considering:

1. Avoid big refactors on `useEffect` and other hooks
2. **Always** put explanation when disabling the rule
3. MobX observers are used with High-Order Components (HOCs)

See example of #3 in this same file